packages/core/package.json

```diff
@@ -18,7 +18,7 @@
     "test:bundles": "source-map-explorer dist/test-bundles/webpack.bundle.js",
     "build": "npm-run-all -s build:tokens build:sass build:ts build:sass:min build:package build:api build:copy",
     "build:watch": "npm-run-all -p build:sass:watch build:ts:watch",
-    "build:package": "cpy '**/package.json' '../dist/core/' --cwd=./src --parents && cpy ./package.json ./dist/core/ && cpy ./README.md ./dist/core/; replace '0.0.0' $npm_package_version ./dist/core/package.json; replace '@VERSION' $npm_package_version ./dist/core/internal/utils/global.js && node ./package.js",
+    "build:package": "cpy '**/package.json' '../dist/core/' --cwd=./src --parents && cpy ./package.json ./dist/core/ && cpy ./README.md ./dist/core/; replace '@VERSION' $npm_package_version ./dist/core/internal/utils/global.js && node ./package.js",
```

Comment:
Revert this file, we need to talk about this since Core doesn't follow the same pattern as others and it shouldn't be part of this change.

Reply:
Done